src/Stratis.Bitcoin.Features.Api/ApiSettings.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.Api
+{
+    /// <summary>
+    /// Configuration related to the API interface.
+    /// </summary>
+    public class ApiSettings
+    {
+        /// <summary>URI to node's API interface.</summary>
+        public Uri ApiUri { get; set; }
+
+        /// <summary>The callback used to override/constrain/extend the settings provided by the Load method.</summary>
+        private Action<ApiSettings> callback = null;
```

Comment:
I don't think this `= null` is necessary.

Reply:
Removed.